natlas-server/app/admin/routes.py

```diff
@@ -396,6 +392,7 @@ def deleteScript(name):
 @isAuthenticated
 @isAdmin
 def deleteScan(scan_id):
+	# TODO - This function redirects you to the deleted scan, which temporarily works because it's an async delete call.
```

Comment:
Its better practice to create an issue/story for this with the details of what you want to fix.